### ISSUE TITLE
ChatGPT Request Not Initiating on Other Browser Pages or Directly from background.js

### ISSUE THREAD

**Issue by giridhar196** [kudoai/chatgpt.js#150]
After adding the starter pack to my website, I encountered an issue. When I restart the extension and refresh the browser page, the request is not being made until I navigate to the chat.open.ai page. Is there a solution to call ChatGPT in the background from other pages?

**Comment by adamlui** [comment]
sup @giridhar196 currently it only works on chat.openai.com but it is definitely possible to call ChatGPT on other pages, just I didn't add those methods to chatgpt.js yet (but you can see how i do it in https://github.com/KudoAI/duckduckgpt e.g. `getShowReply()`

**Comment by adamlui** [comment]
@giridhar196 yes but I am severely afflicted with COVID (for years now) so most of my time is spent in bed recovering but please be patient!

**Comment by adamlui** [comment]
hey @giridhar196 are you still interested? If so, I just integrated OpenRouter API key support (you can grab free key @ https://openrouter.ai) and I could teach you what to do

**Comment by giridhar196** [comment]
No worries I moved away. Thanks
